CHAT_BYLAWS.md

```diff
@@ -0,0 +1,49 @@
+# Apache Polaris - Project Chat Bylaws
```

Comment:
I would include ASF header here (just to be clean as packaged in the distribution).